opencl-aot/source/utils.cpp

```diff
@@ -179,7 +179,7 @@ getOpenCLPlatform(DeviceType Type) {
   std::string PlatformName;
 
   const cl_uint MaxPlatformsCount = 10;
```

Comment:
Now it is not max, right?
Suggest removing the var and create `Platforms` without size passed.

Reply:
It is still max. On line 186 below,

- `MaxPlatformsCount` must be specified when querying for available platforms;
- `Platforms.data()` array must provide `MaxPlatformsCount` elements;
- `PlatformsCount` cannot be > `MaxPlatformsCount`.

Reply:
Removed on second thought.